builder/vsphere/driver/vm.go

```diff
@@ -69,6 +66,13 @@ type CreateConfig struct {
 	USBController bool
 	Version       uint   // example: 10
 	Firmware      string // efi or bios
+	Storage       []Disk
```

Comment:
I was planning on adding the acceptance test as part of the removal of the old fields. I'm not sure the acceptance tests are even running at the moment.

Reply:
I created issue #8841 to get the acceptance test conversation going.

Reply:
@jhawk28 thanks for making the changes and creating the acceptance test issue; apologies for the delayed response here. 

In speaking with the team we are going to let this PR sit for a bit until we can get the acceptance tests running. The changes look good as they are but we want to make sure we are properly setup to test new additions to the code base before merging in enhancements or major fixes. 

I know this might delay the additional changes you have in mind, but it helps us in ensuring things are working as good, if not better, than they are now. Thanks again for pushing through this change and for sticking with us as we get the acceptance testing squared away.